membership-attribute-service/conf/touchpoint.DEV.conf

```diff
@@ -2,6 +2,7 @@ touchpoint.backend.environments {
   DEV {
     dynamodb.table = MembershipAttributes-DEV
     behaviour.dynamodb.table = MembershipBehaviour-DEV
+    featureToggles.dynamodb.table = MembershipFeatureToggles-DEV
```

Comment:
dynamodb is a nice way to do this as it's easy to edit in AWS console compared with S3.